app/text-answer-component/text-answer.css

```diff
@@ -0,0 +1,3 @@
+button {
```

Comment:
Always always when you're styling components, style them with rules like:
```
text-answer button {
}
```
and never just  
```
button {

}
```

The latter will set styles for *every button in the app*, whether it's in your component or not! And you know that the picture-answer folks will be very unhappy if you do that.